tests/utils/test_common.py

```diff
@@ -159,38 +160,63 @@ def test_cli_missing_log_level_default_used():
     configure_logging_and_warnings()
     rasa_logger = logging.getLogger("rasa")
     # Default log level is currently INFO
-    rasa_logger.level == logging.INFO
+    assert rasa_logger.level == logging.INFO
     matplotlib_logger = logging.getLogger("matplotlib")
     # Default log level for libraries is currently ERROR
-    matplotlib_logger.level == logging.ERROR
+    assert matplotlib_logger.level == logging.ERROR
 
 
 def test_cli_log_level_debug_used():
     """Test CLI with log level uses for rasa logger whereas libraries stay default."""
     configure_logging_and_warnings(logging.DEBUG)
     rasa_logger = logging.getLogger("rasa")
-    rasa_logger.level == logging.DEBUG
+    assert rasa_logger.level == logging.DEBUG
     matplotlib_logger = logging.getLogger("matplotlib")
     # Default log level for libraries is currently ERROR
-    matplotlib_logger.level == logging.ERROR
+    assert matplotlib_logger.level == logging.ERROR
 
 
 @mock.patch.dict(os.environ, {"LOG_LEVEL": "WARNING"})
 def test_cli_log_level_overrides_env_var_used():
     """Test CLI log level has precedence over env var."""
     configure_logging_and_warnings(logging.DEBUG)
     rasa_logger = logging.getLogger("rasa")
-    rasa_logger.level == logging.DEBUG
+    assert rasa_logger.level == logging.DEBUG
     matplotlib_logger = logging.getLogger("matplotlib")
     # Default log level for libraries is currently ERROR
-    matplotlib_logger.level == logging.ERROR
+    assert matplotlib_logger.level == logging.ERROR
 
 
 @mock.patch.dict(os.environ, {"LOG_LEVEL": "WARNING", "LOG_LEVEL_MATPLOTLIB": "INFO"})
 def test_cli_missing_log_level_env_var_used():
     """Test CLI without log level uses env var for both rasa and libraries."""
     configure_logging_and_warnings()
     rasa_logger = logging.getLogger("rasa")
-    rasa_logger.level == logging.WARNING
+    assert rasa_logger.level == logging.WARNING
     matplotlib_logger = logging.getLogger("matplotlib")
-    matplotlib_logger.level == logging.INFO
+    assert matplotlib_logger.level == logging.INFO
+
+
+def test_cli_logging_configuration() -> None:
```

Comment:
can we add tests for some of the errors mention [here](https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig)? How does the CLI behave in case of a malformed logging config?

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11683/commits/ba2b69cdfbe1953e93792ae1b1e70fc95f53841b